streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java

```diff
@@ -162,4 +164,27 @@ public static Metric getMetricByNameFilterByTags(final Map<MetricName, ? extends
             return metric;
         }
     }
+
+    public static class IntNumberMatcher extends BaseMatcher<Object> {
```

Comment:
I didn't like matching on the exact Double value, and Hamcrest doesn't seem to have a matcher that does something like this...